src/components/TeacherViews/Onboarding/RoleSelection/RoleSelection.view.tsx

```diff
@@ -34,13 +34,15 @@ const RoleSelectionView = () => {
     setValue,
     handleSubmit,
     clearErrors,
-    setError,
     getValues,
     control,
     trigger,
   } = useForm<RoleSelectFormProps>({
     resolver: zodResolver(roleSelectFormSchema),
     mode: "onBlur",
+    defaultValues: {
+      newsletterSignUp: false,
+    },
```

Comment:
I needed https://github.com/oaknational/Oak-Web-Application/pull/2797 for the `refine` validation to run (it was bailing early in the tests since `newsletterSignUp` was always invalid)